prody/proteins/interactions.py

```diff
@@ -0,0 +1,2043 @@
+        self._interactions_matrix = InteractionsMap
+        
+        return InteractionsMap
+
+
+    def showInteractions(self, **kwargs):
+        """Display protein residues and their number of potential interactions
+        with other residues from protein structure.
+        """
+        
+        import numpy as np
+        import matplotlib
+        import matplotlib.pylab as plt        
+        from prody.dynamics.plotting import showAtomicLines
+        
+        if not hasattr(self, '_interactions_matrix') or self._interactions_matrix is None:
+            raise ValueError('Please calculate interactions matrix first')
+
+        interaction_matrix = self._interactions_matrix
+        atoms = self._atoms 
+        
+        freq_contacts_residues = np.sum(interaction_matrix, axis=0)
+        ResNumb = atoms.select('protein and name CA').getResnums()
+        ResName = atoms.select('protein and name CA').getResnames()
+        ResChid = atoms.select('protein and name CA').getChids()
+
+        ResList = [ i[0]+str(i[1])+i[2] for i in list(zip(ResName, ResNumb, ResChid)) ]
+        
+        matplotlib.rcParams['font.size'] = '20' 
+        fig = plt.figure(num=None, figsize=(12,6), facecolor='w')
+        showAtomicLines(freq_contacts_residues, atoms=atoms.select('name CA'), **kwargs)
+        plt.ylabel('Score of interactions')
+        plt.xlabel('Residue')
+        plt.tight_layout()
+        plt.show()
+        
+    
+    def saveInteractionsPDB(self, **kwargs):
+        """Save the number of potential interactions to PDB file in occupancy column.
+        
+        :arg output: name of the PDB file which will be saved for visualization,
+                     it will contain the results in occupancy column.
+        :type output: str
+        """
+        
+        if not hasattr(self, '_interactions_matrix') or self._interactions_matrix is None:
+            raise ValueError('Please calculate interactions matrix first')
+
+        import numpy as np
+        interaction_matrix = self._interactions_matrix
+        atoms = self._atoms     
+        freq_contacts_residues = np.sum(interaction_matrix, axis=0)
+        
+        try:
+            from collections import Counter
+            lista_ext = []
+            atoms = atoms.select("all and noh")
+            aa_counter = Counter(atoms.getResindices())
+            calphas = atoms.select('name CA')
+            for i in range(calphas.numAtoms()):
+                lista_ext.extend(list(aa_counter.values())[i]*[round(freq_contacts_residues[i], 8)])
+            
+            kw = {'occupancy': lista_ext}
+            if 'output' in kwargs:
+                writePDB(kwargs['output'], atoms, **kw)  
+                LOGGER.info('PDB file saved.')
+            else:
+                writePDB('output', atoms, **kw)
+                LOGGER.info('PDB file saved.')
+        except: LOGGER.info('There is a problem.')
+        
+
+    def getFrequentInteractions(self, contacts_min=3):
```

Comment:
perhaps this could say frequent interactors instead

Reply:
FrequentInteractions instead of getFrequentInteractions, or you mean contact_min to frequent_interactions?

Reply:
no. Like we discussed with Ivet, its the residues and not the interactions that are frequent and therefore we need a different name for the function. My suggestion was getFrequentInteractors with an ORS and not getFrequentInteractions with an IONS

Reply:
Ok. Now I see. Thanks.

Reply:
You're welcome